src/lib/db/api-token-store.ts

```diff
@@ -270,4 +271,43 @@ export class ApiTokenStore implements IApiTokenStore {
             this.logger.error('Could not update lastSeen, error: ', err);
         }
     }
+
+    async countDeprecatedTokens(): Promise<{
+        orphanedTokens: number;
+        activeOrphanedTokens: number;
+        legacyTokens: number;
+        activeLegacyTokens: number;
+    }> {
+        const allLegacy = await this.makeTokenProjectQuery()
+            .whereNull('token_project_link.project')
+            .andWhere('tokens.secret', 'NOT LIKE', '%:%');
+
+        const activeLegacy = await this.makeTokenProjectQuery()
```

Comment:
can we run those queries in parallel instead of sequentially?

Reply:
or maybe we can even join all those queries into one count query similar to what you did in createGetActiveUsers with CASE WHEN?